checker/checker_test.go

```diff
@@ -21,7 +22,9 @@ func cleanup(value string) string {
 	return result
 }
 
-func TestCompile(t *testing.T) {
+func TestChecker_Check(t *testing.T) {
+	t.Parallel()
```

Comment:
`t.Parallel()` should be inside the `t.Run`

Reply:
The whole test is about 10ms with most of the cases in under 1 ms. Having the parallel at the test case level mangles the output and makes it more difficult to debug. Specially since you cannot just run a specific case.